commit 36f441ba50af516079ceaa888164903bc1ce881f
Author: Volker Krause <vkrause@kde.org>
Date:   Mon Jan 1 18:35:55 2018 +0100

    Make kdoctools dependency optional
    
    Same as in other frameworks, makes cross-compiling easier.

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1ed87a14..7826413d 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -28,7 +28,7 @@ find_package(KF5ConfigWidgets ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5Crash ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5DesignerPlugin ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5GlobalAccel ${KF5_DEP_VERSION} REQUIRED)
-find_package(KF5DocTools ${KF5_DEP_VERSION} REQUIRED)
+find_package(KF5DocTools ${KF5_DEP_VERSION})
 find_package(KF5Emoticons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5GuiAddons ${KF5_DEP_VERSION} REQUIRED)
 find_package(KF5I18n ${KF5_DEP_VERSION} REQUIRED)
@@ -153,11 +153,15 @@ install(
 
 if (IS_DIRECTORY "${CMAKE_CURRENT_SOURCE_DIR}/po")
     ki18n_install(po)
-    kdoctools_install(po)
+    if (KF5DocTools_FOUND)
+        kdoctools_install(po)
+    endif()
 endif()
 add_subdirectory(cmake)
 add_subdirectory(data)
-add_subdirectory(docs)
+if (KF5DocTools_FOUND)
+    add_subdirectory(docs)
+endif()
 add_subdirectory(src)
 
 if (BUILD_TESTING)
diff --git a/KF5KDELibs4SupportConfig.cmake.in b/KF5KDELibs4SupportConfig.cmake.in
index b5194f94..cf3f2aca 100644
--- a/KF5KDELibs4SupportConfig.cmake.in
+++ b/KF5KDELibs4SupportConfig.cmake.in
@@ -12,7 +12,9 @@ find_dependency(KF5ConfigWidgets "@KF5_DEP_VERSION@")
 find_dependency(KF5CoreAddons "@KF5_DEP_VERSION@")
 find_dependency(KF5Crash "@KF5_DEP_VERSION@")
 find_dependency(KF5DesignerPlugin "@KF5_DEP_VERSION@")
-find_dependency(KF5DocTools "@KF5_DEP_VERSION@")
+if(@KF5DocTools_FOUND@)
+    find_dependency(KF5DocTools "@KF5_DEP_VERSION@")
+endif()
 find_dependency(KF5Emoticons "@KF5_DEP_VERSION@")
 find_dependency(KF5GuiAddons "@KF5_DEP_VERSION@")
 find_dependency(KF5IconThemes "@KF5_DEP_VERSION@")
diff --git a/cmake/modules/KDE4Macros.cmake b/cmake/modules/KDE4Macros.cmake
index ca868ad9..5c50a2cf 100644
--- a/cmake/modules/KDE4Macros.cmake
+++ b/cmake/modules/KDE4Macros.cmake
@@ -485,7 +485,9 @@ endmacro (KDE4_ADD_LIBRARY _target_NAME _lib_TYPE)
 include("${KF5Auth_DIR}/KF5AuthMacros.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/KDECoreMacros.cmake")
 include("${CMAKE_CURRENT_LIST_DIR}/KDEUIMacros.cmake")
-include("${KF5DocTools_DIR}/KF5DocToolsMacros.cmake")
+if (KF5DocTools_FOUND)
+    include("${KF5DocTools_DIR}/KF5DocToolsMacros.cmake")
+endif()
 include("${KF5DesignerPlugin_DIR}/KF5DesignerPluginMacros.cmake")
 
 # functions deprecated, they still should work
